code/datums/shuttles.dm

```diff
@@ -140,6 +140,16 @@
 	description = "A Tadpole model for hauling, engineering and general maintenance. Patented by Nakamura Engineering, and is a rather reliable way to transport goods."
 	display_name = "Tadpole NK-Haul Model"
 
+/datum/map_template/shuttle/minidropship/mobile_bar
+    suffix = "_mobile_bar"
+    description = "A Tadpole modified to provide foods and services. God damit its him again I thought we got rid of him."
+    display_name = "Tadpole Mobile-Bar Model"
+
+/datum/map_template/shuttle/minidropship/combat_tad
+    suffix = "_combat_tad"
+    description = "A Tadpole modified to have three hardpoints instead of just one, the majority of the other standerd featured had to be scrapped to fit all three of them on. Also comes equiped with 2 electric hardpoints"
```

Comment:
```suggestion
    description = "A Tadpole model modified to have three weapon hardpoints instead of just one, the majority of the other standard features had to be scrapped to fit all three of them on. Also comes equipped with 2 electric hardpoints."
```